tests/common/plugins/memory_utilization/README.md

```diff
@@ -0,0 +1,88 @@
+### Scope
+This document provides a high level description of the memory utilization verification design and instructions on using the "memory_utilization" fixture in SONiC testing.
+
+### Overview
+During testing, the memory usage of the DUT can vary due to different configurations, environment setups, and test steps. To ensure the safe use of memory resources, it is necessary to check the memory usage after the test to confirm that it has not exceeded the high memory usage threshold and that no memory leaks have occurred.
+
+The purpose of the current feature is to verify that memory resources do not increase during test runs and do not exceed the high memory usage threshold.
+
+### Module Design
+Newly introduced a plugin "memory_utilization" and config files.
+
+#### Config files
+Config files, including common file and platform dependence file.
+- **memory_utilization_common.json**: Common configurations for memory utilization.
+- **memory_utilization_dependence.json**: Dependency configruations for memory utilization, currently supporting special configurations for specific HwSku devices. This configuration file should be left empty in the public branch, with special configurations added in internal branch.
```

Comment:
@lipxu  could we give detailed example about the hwsku configuration?
could we have customized check per testcase?

Reply:
Sure, had updated the README, and add the example for hwsku configuration